SEED-platform/seed
# 855 enforce spelling

#### Any background context you want to provide?
When we cleaned up the documentation we made sure that everything was spelled correctly. 

#### What's this PR do?
This will enforce spell check in docstrings and rst files.

#### How should this be manually tested?
Locally run cd docs && make spelling. Fix errors. Note that for some reason Travis is more stringent on some words. So you may have to add more words to the spelling_wordlist.txt

#### What are the relevant tickets?
Refs: #855 

#### Definition of Done:
- [X] Is there appropriate test coverage? Yes
- [X] Does this PR require a Selenium test? No
- [X] Does this PR require a regression test? No
- [X] Does this add new dependencies? Yes, enchant, If so, does PIP, npm, bower requirements need to be updated? Yes.
